.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@aperdomob @germandavid85 @luigisamurai @yesidbalvin @Scot3004 @santirogu
```

Comment:
You missed the `* ` at the beginning of the line, also, @Valeryibarra @leonleo997 @renardete and me (@holgiosalos) should be the only ones on this file